### ISSUE TITLE
Malware domains

### ISSUE THREAD

**Issue by iam-py-test** [DandelionSprout/adfilt#188]
## Which entry/entries are you submitting?

`gogle.net`
`quatrefeuillepolonaise.xyz`

## Which things do they block, hide, or unbreak?

These entries block malware domains.

Verification:
 https://www.siteadvisor.com/sitereport.html?url=gogle.net
 https://www.mywot.com/scorecard/gogle.net
 https://www.urlvoid.com/scan/gogle.net/
 https://www.fortiguard.com/webfilter?q=gogle.net
 https://www.virustotal.com/gui/domain/gogle.net/detection
https://github.com/iam-py-test/my_filters_001/blob/66213bbc1c0cac88356917f30eb07ac569cf8588/antimalware.txt#L33
Found in curl response of `gogle.net`
https://www.siteadvisor.com/sitereport.html?url=quatrefeuillepolonaise.xyz https://www.virustotal.com/gui/url/7319b37aff351dc0f0e71dba194b5f21972be9ad072b955a35d27d5af359d5fa/community https://www.virustotal.com/gui/domain/quatrefeuillepolonaise.xyz/detection
https://safeweb.norton.com/report/show?url=quatrefeuillepolonaise.xyz
https://www.fortiguard.com/webfilter?q=quatrefeuillepolonaise.xyz
https://quttera.com/detailed_report/quatrefeuillepolonaise.xyz
 https://www.urlvoid.com/scan/quatrefeuillepolonaise.xyz/
 https://www.mywot.com/en/scorecard/quatrefeuillepolonaise.xyz

## Which of my lists are you submitting it to?
Antimalware
## Which adblocker(s) and version did you use when writing and testing the entries?

- [X] uBlock Origin
- [ ] AdGuard (Paid desktop version)
- [ ] AdGuard (for browsers)
- [ ] AdBlock
- [ ] Adblock Plus
- [ ] AdGuard Home
- [ ] Blokada
- [ ] I Don't Care About Cookies (The extension)

Other(s):

Adblocker version(s):
uBlock Origin development build v1.35.3b5
## Which filterlists did you use? Failing to tell this will temporarily close the report until it has been told.

![](https://user-images.githubusercontent.com/84232764/119387510-0c74d900-bc97-11eb-8301-b47f7880148b.png)

![](https://user-images.githubusercontent.com/84232764/119387645-37f7c380-bc97-11eb-879f-a6ab45d12e25.png)

![](https://raw.githubusercontent.com/iam-py-test/Assets-001/main/images/Screenshot%202021-05-24%20135147.jpeg)

## (Optional) Which browser(s) and version did you use?

Edge Version 90.0.818.66 (Official build) (64-bit)

**Comment by iam-py-test** [comment]
@DandelionSprout is there other info I need to add?
Also, if I want to report other domains, should I add them to this issue or open a new one?

**Comment by llacb47** [comment]
The first two are fine, the 3rd is a p0rn site, not malware, as uBlock-user explained to you.

**Comment by iam-py-test** [comment]
> The first two are fine, the 3rd is a p0rn site, not malware, as uBlock-user explained to you.

What do you mean the first two are fine?
![image](https://user-images.githubusercontent.com/84232764/119536579-3c83b100-bd57-11eb-9c75-5dc2b8a8cf01.png)
https://safeweb.norton.com/report/show?url=gogle.net
I will remove the porn one.

**Comment by iam-py-test** [comment]
Strangely, it now shows `quatrefeuillepolonaise.xyz` is offline.
Closing issue.
Sorry for wasting your time

**Comment by llacb47** [comment]
Fine to report as malware I meant, sorry for being confusing

**Comment by iam-py-test** [comment]
Using `https://redirectdetective.com` I found that `gogle[.]net` redirects to `hxxp[:]//quatrefeuillepolonaise[.]xyz/?k=51ca53e382eb9fb4091c78523a668bed.1621979609.91.0.1.d3cuZ29nbGUubmV0&r=` which redirects to `hxxps://extragifis[.]site/?u=9618mwc&o=rcbkvzh&t=rp134`

**Comment by iam-py-test** [comment]
https://safeweb.norton.com/report/show_mobile?name=https://extragifis.site/?u=9618mwc&o=rcbkvzh&t=rp134

**Comment by iam-py-test** [comment]
@DandelionSprout Do you have a Virtual Env you can use to investigate?

**Comment by iam-py-test** [comment]
![image](https://user-images.githubusercontent.com/84232764/119574215-5e932880-bd83-11eb-8e6e-ac13f3feb9b7.png)

**Comment by iam-py-test** [comment]
Tried again - no redirect to `extragifis.site`:
![image](https://user-images.githubusercontent.com/84232764/119574765-f4c74e80-bd83-11eb-8f22-d803ba6d95ce.png)

**Comment by iam-py-test** [comment]
Maybe it detected the redirect detective and blocked it?

**Comment by DandelionSprout** [comment]
>Do you have a Virtual Env you can use to investigate?

I do have «Windows Sandbox», which is a whole lot easier to set up than a regular VM. Overall, I am currently on the case of looking into `extragifis`, which is more than a little malicious from what I can see. Please be patient and calm in the meantime.

**Comment by iam-py-test** [comment]
> > Do you have a Virtual Env you can use to investigate?
> 
> I do have «Windows Sandbox», which is a whole lot easier to set up than a regular VM. Overall, I am currently on the case of looking into `extragifis`, which is more than a little malicious from what I can see. Please be patient and calm in the meantime.

Ok! 
Thank you. I should have known you had Windows Sandbox set up by reading past issues

**Comment by iam-py-test** [comment]
@DandelionSprout Thank you!

**Comment by DandelionSprout** [comment]
All the stuff earlier in this thread was a very big mouthful for me to get a grasp of, so I hope it's understandable that I simply added some entries and declared this issue closed now. I also took care of #185 in light of new evidence there.

**Comment by iam-py-test** [comment]
Ok! Thanks

**Comment by iam-py-test** [comment]
What are the other domains you added?

**Comment by DandelionSprout** [comment]
The last two were ones that the `extragifis` link redirected me to.

**Comment by iam-py-test** [comment]
Ok! Thanks for answering my question.
Is it ok if I add those domains to my person antimalware list?

**Comment by DandelionSprout** [comment]
It is of course OK to do so.

**Comment by iam-py-test** [comment]
Thank you!

**Comment by iam-py-test** [comment]
Maybe should add `captcharesolver.com` too.
https://www.virustotal.com/gui/ip-address/5.8.34.26/relations

**Comment by iam-py-test** [comment]
@DandelionSprout Maybe also `chatroom33.com` which resolves to the same IP

**Comment by DandelionSprout** [comment]
I exclusively use https://securitytrails.com for this kind of analysis. It confirms `captcharesolver.com`, but not `chatroom33.com`

**Comment by iam-py-test** [comment]
> I exclusively use https://securitytrails.com for this kind of analysis. It confirms `captcharesolver.com`, but not `chatroom33.com`

https://www.virustotal.com/gui/domain/chatroom33.com/detection
![image](https://user-images.githubusercontent.com/84232764/119577980-a3ba5900-bd89-11eb-8fa4-4aac4e560390.png)

**Comment by iam-py-test** [comment]
Do you think I should report these domains to Google Safe Browsing too so it is blocked for more users
<details>
<summary>Irrelevant</summary>
What are your pronouns?
I would just like to know what pronouns to use when talking to other people
</details>

**Comment by iam-py-test** [comment]
What about https://www.virustotal.com/gui/ip-address/204.11.56.48/community?
https://www.ipvoid.com/ip-blacklist-check/ shows `BLACKLISTED 3/115`

**Comment by iam-py-test** [comment]
Question: Why is this use $all vs $document vs just a hostname?https://github.com/DandelionSprout/adfilt/blob/3b3c6a90981f428f55174c60cdfd97721efc285a/Dandelion%20Sprout's%20Anti-Malware%20List.txt#L517

**Comment by DandelionSprout** [comment]
>`https://www.virustotal.com/gui/domain/chatroom33.com/detection`

My immediate concern is with the domains at `https://securitytrails.com/list/ip/5.8.47.3`, rather than ones at `5.8.34.26`, to be honest.

>What are your pronouns? (…)

Generally he/him, but I'm not like all those people on social media who treat misgendering like a matter of life and death.

>What about `204.11.56.48`?

`https://securitytrails.com/list/ip/204.11.56.48` gives too many results for a broad block to be feasible.

>Question: Why is this use $all vs $document vs just a hostname?

It depends on the severity of the domain:
* `$all`: For sites that should not be visited under **any** circumstances.
* `$doc`: For sites where users need to be asked "Are you *really* sure you want to visit this site?"
* (hostname only): Usually entries that are intended for AdGuard Home and Pi-Hole

**Comment by iam-py-test** [comment]
Ok! Thanks

**Comment by iam-py-test** [comment]
Looks like a lot of porn domains on the first one

**Comment by iam-py-test** [comment]
@DandelionSprout do you know if `cpmstatsart.com` is malicious?
It showed up in the response from `gogle.net` with a spoofed UA.

**Comment by iam-py-test** [comment]
@DandelionSprout `cpmstatsart.com` looks malware to me:
https://www.virustotal.com/gui/url/c7d15fdbb73c179e13c3354a7183e2227034d6a95e9ebe2e312d6dd7ece51ec5/detection
https://www.fortiguard.com/webfilter?q=cpmstatsart.com
https://quttera.com/detailed_report/cpmstatsart.com

**Comment by iam-py-test** [comment]
Should it be added to the list?

**Comment by iam-py-test** [comment]
@DandelionSprout Requested the `cpmstatsart.com` url (which is an iframe) and uploaded response
https://github.com/iam-py-test/Assets-001/blob/main/moremalwarefromgoglenet_related

**Comment by iam-py-test** [comment]
Looks like a spammy/malvertizing ad

**Comment by iam-py-test** [comment]
I was able to get these stats on it:
```Enter the host to get info on:cpmstatsart.com
Final response url: http://cpmstatsart.com/
No redirects
cpmstatsart.com has not set a 'Strict-Transport-Security' header
SHA-256 of response:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
Error:list index out of range  No meta refresh found
No noscripts:[]
Without redirects, returns status 200
URLHaus detected as not malware
PhishingFilter deteced as not phishing
Ranking: -1
Host is not spam
Notespad did not detect cpmstatsart.com as malware or phishing
whois domain name: cpmstatsart.com
whois registrar: Above.com Pty Ltd.
whois creation date: 2017-08-21 08:41:22
Host ip: 103.224.182.251
Main page can be crawled:True
```

**Comment by iam-py-test** [comment]
Not sure if they are helpful but thought they might

**Comment by iam-py-test** [comment]
Same stats on `gogle.net`:
``` Enter the host to get info on:gogle.net
Final response url: http://gogle.net/
No redirects
gogle.net has not set a 'Strict-Transport-Security' header
SHA-256 of response:e30ac3fd5ce81b4ecc9f7db4bb27acb40dba0760e559a86a120ca89df4f5d8be
0;url=http://quatrefeuillepolonaise.xyz/?k=c5cb34abfd93a1e8aa94282acba8e09b.1622038774.806.0.1.d3cuZ29nbGUubmV0&r=
Noscript found:
Without redirects, returns status 200
URLHaus detected as not malware
PhishingFilter deteced as not phishing
Ranking: -1
Host is not spam
Notespad detected gogle.net as malware or phishing
Notespad detected the final url as malware or phishing
whois domain name: gogle.net
whois registrar: Gransy, s.r.o.
whois creation date: 2010-06-13 18:14:33
Host ip: 54.235.212.68
Main page can be crawled:True
```

**Comment by iam-py-test** [comment]
The `0;url=http://quatrefeuillepolonaise.xyz/?k=c5cb34abfd93a1e8aa94282acba8e09b.1622038774.806.0.1.d3cuZ29nbGUubmV0&r=` is a meta refresh it found

**Comment by iam-py-test** [comment]
Same stats on `quatrefeuillepolonaise.xyz`:
```
Enter the host to get info on:quatrefeuillepolonaise.xyz

Can not reach quatrefeuillepolonaise.xyz
Can not reach
URLHaus detected as not malware
PhishingFilter deteced as not phishing
Ranking: 144400
Host is not spam
Notespad did not detect quatrefeuillepolonaise.xyz as malware or phishing
whois domain name: quatrefeuillepolonaise.xyz
whois registrar: Namecheap
whois creation date: 2020-04-03 11:09:05
Host ip: 34.196.13.28
Main page can be crawled:True
 ```

**Comment by iam-py-test** [comment]
Don't know why it shows `can not reach `

**Comment by iam-py-test** [comment]
Ranking is from Tranco

**Comment by iam-py-test** [comment]
Retried again:
```
Can not reach quatrefeuillepolonaise.xyz
Can not reach
URLHaus detected as not malware
PhishingFilter deteced as not phishing
Ranking: 144400
Host is not spam
Notespad did not detect quatrefeuillepolonaise.xyz as malware or phishing
whois domain name: quatrefeuillepolonaise.xyz
whois registrar: Namecheap
whois creation date: 2020-04-03 11:09:05
Host ip: 34.196.13.28
Main page can be crawled:True
```
I think it is blocking my crawler. Will spoof its UA and hope it has not added an IP level block.

**Comment by iam-py-test** [comment]
Ok! Sorry

**Comment by iam-py-test** [comment]
> Can you edit your comment instead of making 10 different comments? And only post important/relevant stuff. This is not a chatroom.

Is that better? Sorry I posted relevant comments

**Comment by iam-py-test** [comment]
Another malware domain `instantfwding.com`
https://www.virustotal.com/gui/url/ad0c37cbc87adfb19706855150eefc0dcacff563d50cbf653ae46f48906e1dc7/detection

**Comment by iam-py-test** [comment]
@DandelionSprout Thanks!

**Comment by iam-py-test** [comment]
@DandelionSprout `gamesex.fun` and `goldprize.xyz` from https://securitytrails.com/list/ip/5.8.47.3  also look malware.
https://www.virustotal.com/gui/domain/goldprize.xyz/detection
https://safeweb.norton.com/report/show?url=goldprize.xyz

**Comment by iam-py-test** [comment]
Another domain resolving to a `gogle.net`-related ip: `gmailgmail.com`.
It _also_ loads scripts from the same domain: `quatrefeuillepolonaise.xyz`
https://www.virustotal.com/gui/url/6258b848548d5341b4f05b53b7fcbb46a6aa751d306de43f46a081bd94e6271d/detection
https://www.virustotal.com/gui/domain/gmailgmail.com/detection
https://www.fortiguard.com/webfilter?q=gmailgmail.com
https://www.siteadvisor.com/sitereport.html?url=gmailgmail.com
https://www.urlvoid.com/scan/gmailgmail.com/
https://www.mywot.com/en/scorecard/gmailgmail.com
https://safeweb.norton.com/report/show?url=gmailgmail.com
https://quttera.com/detailed_report/gmailgmail.com
https://sitecheck.sucuri.net/results/gmailgmail.com

Same as above: `gmsail.com`
https://www.virustotal.com/gui/url/8333708782cbe562f064b5dbb13fee9d63a571c3d3e283c6b359cd79010768c5/detection
https://www.virustotal.com/gui/url/20aed8a116ee3d494e24635d01ffd01c3c3a76a693c40613afcf706a0a61e0ec/detection
https://www.fortiguard.com/webfilter?q=gmsail.com
https://www.urlvoid.com/scan/gmsail.com/
https://www.mywot.com/en/scorecard/gmsail.com
https://safeweb.norton.com/report/show?url=gmsail.com
https://sitecheck.sucuri.net/results/gmsail.com
https://quttera.com/detailed_report/gmsail.com
https://www.siteadvisor.com/sitereport.html?url=gmsail.com (it might think it is parked because it redirects to `quatrefeuillepolonaise.xyz:`)
Currently these domains homepages appear to redirect to `quatrefeuillepolonaise.xyz` but there could be other pages which do something else plus it is very likely they will never be used for legit purposes.  (apart from being parked)

**Comment by iam-py-test** [comment]
Another domain like above: `homal.com`
https://www.virustotal.com/gui/url/cbfb94d5fd5efb4bc705d686d9c119e9ef44e630273f073a9b7dff6250a3b2b8/detection
https://www.virustotal.com/gui/domain/homal.com/detection
https://www.virustotal.com/gui/url/a325e15dd9182d37c811bb8fbc1691b24bc6f45c6600e1af868c461ef9787fb1/detection
https://www.fortiguard.com/webfilter?q=homal.com
https://www.urlvoid.com/scan/homal.com/
https://www.mywot.com/en/scorecard/homal.com
https://safeweb.norton.com/report/show?url=homal.com
https://quttera.com/detailed_report/homal.com
https://sitecheck.sucuri.net/results/homal.com


Also redirects to `quatrefeuillepolonaise.xyz` but has a history of being malware and spam.
I am not sure if you want to spend a lot of time blocking domains that just redirect to another blocked domain.

**Comment by DandelionSprout** [comment]
I am not 100% sure when I'll be able to go through the backlog, because I've also got a somewhat big backlog over at https://github.com/DandelionSprout/adfilt/discussions/163 to take care of sometime later today.

**Comment by iam-py-test** [comment]
> I am not 100% sure when I'll be able to go through this backlog, because I've also got a somewhat big backlog over at #163 to take care of sometime later today.

Sorry for posting so many comments. I have deleted the ones that just redirect to another domain

**Comment by iam-py-test** [comment]
Found in community reports on https://www.virustotal.com/gui/ip-address/103.224.182.251/community: <br/> `www.iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com`
https://www.virustotal.com/gui/url/d6b49bc9d8b59af67edcac2f87ef4a63bc0d6760d03076decb79e413dbf61ae5/detection
https://www.virustotal.com/gui/domain/www.iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com/detection
https://www.joesandbox.com/analysis/368427/0/html
https://www.virustotal.com/gui/domain/www.iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com/detection
https://www.virustotal.com/gui/domain/iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com/detection

The community reports on this domain go back for years. According to the community reports, this site spreads the WANNACRY ransomware. 

https://www.siteadvisor.com/sitereport.html?url=www.iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com
https://www.fortiguard.com/webfilter?q=www.iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com
https://www.urlvoid.com/scan/iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com/
https://www.mywot.com/en/scorecard/www.iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com
https://safeweb.norton.com/report/show?url=www.iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com
https://sitecheck.sucuri.net/results/www.iuqerfsodp9ifjaposdfjhgosurijfaewrwergwff.com

**Comment by iam-py-test** [comment]
Reported these domains to Google Safe Browsing

**Comment by iam-py-test** [comment]
@DandelionSprout It looks like someone registered a variant on `captcharesolver.com` -  `top-captcharesolver.com` - 3 days ago.

**Comment by spirillen** [comment]
> > Do you have a Virtual Env you can use to investigate?
> 
> I do have «Windows Sandbox», which is a whole lot easier to set up than a regular VM. Overall, I am currently on the case of looking into `extragifis`, which is more than a little malicious from what I can see. Please be patient and calm in the meantime.

I would really like to know what you might have found about `extragifis.site`. As for todays searches the only indication I have found is a html page with `site under construction`

**Comment by iam-py-test** [comment]
> 
> 
> > > Do you have a Virtual Env you can use to investigate?
> > 
> > 
> > I do have «Windows Sandbox», which is a whole lot easier to set up than a regular VM. Overall, I am currently on the case of looking into `extragifis`, which is more than a little malicious from what I can see. Please be patient and calm in the meantime.
> 
> I would really like to know what you might have found about `extragifis.site`. As for todays searches the only indication I have found is a html page with `site under construction`

You will have to ask @DandelionSprout as I think he added that filter (I just copypasted it)
Also, there could be other pages on that site that are malware

**Comment by DandelionSprout** [comment]
I am very certain it was a redirection domain on the day I tested it.

And as long as it's parked or "under construction", there's a high chance that it still is a redirection domain under some circumstances.

**Comment by spirillen** [comment]
That's true @DandelionSprout 

However, I'll think I'll leave the domain at this, as there are no current traces of it being used

![image](https://user-images.githubusercontent.com/44526987/124349020-61d4cc00-dbed-11eb-94aa-3fe20d1744d5.png)

**Comment by iam-py-test** [comment]
@DandelionSprout some new domains which I found when visiting `gogle[.]net` in a VM:
```
retailproductsusa.com
cloud-repos.store
! already blocked by .top
nextarrangewent.top
smartcaptchasolve.top
```

**Comment by spirillen** [comment]
Hey @iam-py-test 

I would really like to know how you get these redirects, you mentioned them here too.

https://mypdns.org/my-privacy-dns/matrix/-/issues/1883

If you in any way can get some headers or other cool logs showing what is happening would be cool.

**Comment by iam-py-test** [comment]
> 
> 
> Hey @iam-py-test
> 
> I would really like to know how you get these redirects, you mentioned them here too.
> 
> https://mypdns.org/my-privacy-dns/matrix/-/issues/1883
> 
> If you in any way can get some headers or other cool logs showing what is happening would be cool.

Sorry; I can not provide logs.
I can try to go back and will record

**Comment by DandelionSprout** [comment]
(To spirillen in particular) From what I understand, iam-py-test is testing in real-time, to look for redirections that sometimes may not even last half a second before the next redirection.

**Comment by iam-py-test** [comment]
Also, Imre is right. I often can't get all of them
Recording: uploading...

**Comment by iam-py-test** [comment]
@DandelionSprout wait.
There are more.....

**Comment by spirillen** [comment]
> (To spirillen in particular) From what I understand, iam-py-test is testing in real-time, to look for redirections that sometimes may not even last half a second before the next redirection.

That's true @DandelionSprout And to be honest I tried to follow the same pattern a couple of years ago, but with no success.

Is you was using  a browser you some times get a redirect, but when you are using curl, lynx or any other text based "browser", the redirect never happens.

My question to @iam-py-test relays to his VM's where he might have installed some logging tools to catch these incidences.

There is nothing new in any of this action, only the targets

The next thing is to figure out what the real purpose of this is, redirecting, ads, tracking, how should be categorize this correctly?

**Comment by iam-py-test** [comment]
@spirillen watch my recording (here - https://iam-py-test.github.io/Data/recording1.zip)
These are clearly malware

**Comment by iam-py-test** [comment]
I can try and use Wireshark...

**Comment by spirillen** [comment]
Wireshark will only intercept the network traffic as far I know, it is the HTTP code from the server response header or any other HTML redirects code we need to figure out what actually happens.

But I'm always happy to be proven wrong... yeah weird right, but I will learn something new those days

**Comment by iam-py-test** [comment]
I can write a browser extension to log that, but it would take some time.

**Comment by spirillen** [comment]
That would be exceptional for many people to have a tool that can record such things :)


What I can't understand in all of this is why snoogle would put there name to this malware?

**Comment by iam-py-test** [comment]
Logging all requests...
Will post log

**Comment by iam-py-test** [comment]
CC @DandelionSprout @blocklistproject

**Comment by spirillen** [comment]
the first ~46 lines looks interesting for further investigation :+1: will have to sort them by domains for my case.

Will follow up on them tomorrow if you haven't created the MR on mypdns before me :+1: 

Thanks for your good work @iam-py-test

**Comment by iam-py-test** [comment]
I will try to upload the tool I create to GitHub just encase you find it helpful (right now its less complex then the uBo logger, but I am thinking of adding more features)

I will be busy most of tomorrow, so probably won't be able to open a MR.

**Comment by DandelionSprout** [comment]
The list contained a fair few legitimate domains, including `Facebook` and `YouTube`, as well as ones that would already be blocked by EasyList (incl. `doubleclick` and `addthis`). This means that I simply cannot use that list verbatim, but there are approx. 30 domains that are more than interesting for me to add. Give me 15min.

**Comment by iam-py-test** [comment]
I am working on filtering it

**Comment by iam-py-test** [comment]
Filtered and removed domains I am sure are legit:
[domains_file.txt](https://github.com/DandelionSprout/adfilt/files/7262889/domains_file.txt)

**Comment by DandelionSprout** [comment]
I'll add those additional entries sometime today (Friday).

**Comment by iam-py-test** [comment]
I will try to blocklist in my lists as well.
I can open a PR maybe tomorrow...
> Though you kinda forgot to remove the MSN domains. 😅

I think that was fixed in the second one

**Comment by DandelionSprout** [comment]
Ah, okay. They had not been removed at the time https://github.com/DandelionSprout/adfilt/issues/188#issuecomment-931752781 was posted.

**Comment by iam-py-test** [comment]
> 
> 
> Ah, okay. They had not been removed at the time [#188 (comment)](https://github.com/DandelionSprout/adfilt/issues/188#issuecomment-931752781) was initially posted.

I am aware. I check it to make sure it uploaded correctly and saw that

**Comment by spirillen** [comment]
That not all of it. some of them are obfuscated as parked/hijacked domains, just take `cpmstatsart.com`usually i would place that as a Hijacked (sharked) domain, not it is also used for scamming trough functioning as ? haven't decoded yet

`http://cpmstatsart.com/mnz/v1?placement=d52af548-a561-11e7-9ab5-02c1c5ed83e8&source=134`

it a lot of work you have posted there...

**Comment by iam-py-test** [comment]
I haven't checked, but I'd bet a lot of these are not detected by many AVs
We should do a mass report to AdGuard/uAssets
CC @blocklistproject @Yuki2718

**Comment by iam-py-test** [comment]
Even more domains
[gogle_net_domains.txt](https://github.com/DandelionSprout/adfilt/files/7270467/gogle_net_domains.txt)
